tests/time_tests/src/timetests_helper/main.cpp

```diff
@@ -40,7 +41,7 @@ bool parseAndCheckCommandLine(int argc, char **argv) {
  */
 int _runPipeline() {
   SCOPED_TIMER(full_run);
-  return runPipeline(FLAGS_m, FLAGS_d, FLAGS_c);
+  return runPipeline(FLAGS_m, FLAGS_d, FLAGS_c, FLAGS_reshape_shapes, FLAGS_data_shapes);
```

Comment:
1. parse values before passing it to execution pipeline
2. handle exceptions during parsing the correct way - provide additional messages about what was wrong